src/messages/nl.json

```diff
@@ -1,6 +1,6 @@
 {
   "Meta": {
-    "title": "Dataset Browser"
+    "title": "Dataset browser"
```

Comment:
I've always considered `Dataset Browser` to be the product name - in which a capital `B` fits. If that's not the case, I think the translation should be `Datasetbrowser`.

Reply:
I changed this because we used two styles, and "Dataset browser" was used in the design, so I chose that one. But I do not know which one to use.

Reply:
Discussed with Gertjan, we will use "Dataset Browser". I will change it.